naucse/templates/index.html

```diff
@@ -9,28 +9,28 @@
     <div class="card card-inverse space motivation">
       <img class="card-img" src="{{url_for('static', filename='pycon2016cz.png')}}" alt="Card image">
       <div class="card-img-overlay">
-        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a zachraň svět.<br>
-            Python je moderní programovací jazyk. Je univerzální – pohání weby i rakety.
-            Dobře se čte a dá se velice rychle naučit.</p>
+        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a&nbsp;zachraň svět.<br>
```

Comment:
I disagree about `&nbsp;`. The reasons make more sense for long text, but I think this should be consistent.
Naucse should ultimately be a site for collaboratively written text. And just like a book has an author and an editor, the people writing (and reading!) the text shouldn't need to concern themselves with typography if they don't want to. If you don't have your editor set up to highlight whitespace, you probably don't care, and that's okay. But having `&nbsp;` everywhere makes the text is unreadable, and I do want people to be able read the source – it makes little contributions easier. 
That all could be „solved“ by an automatic Texy!-like system, but I think keeping things explicit is better. What I'd like to set up, eventually, is a linter – something that flags mistakes on pull requests, so either the author or the person merging it can fix these little mistakes. I don't want a typographical Autocomplete.

About the `a` and `i`: this is not a paragraph of running text. I wouldn't mind putting non-breaking spaces even between long words if that would make it look better as a whole.